benchmarks/AuraDiBench.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace PhpBench\Benchmarks\Container;
+
+use Aura\Di\ContainerBuilder;
+
+/**
+ * @Groups({"aura-di"}, extend=true)
+ */
+class AuraDiBench extends ContainerBenchCase
+{
+    private $container;
+
+    public function benchGetUnoptimized()
+    {
+        $this->container->get('bicycle_factory');
+    }
+
+    public function benchGetOptimized()
+    {
+        $this->container->get('bicycle_factory');
```

Comment:
The thinking was that to assume containers are optimized (i.e. AuraDI has no "un-optimized" state, its standard configuration is "optimized"). But we should change the terminology as it is confusing.

But at least for now it should be `Unoptimized` that is skipped.